airbyte-cdk/python/unit_tests/sources/test_abstract_source.py

```diff
@@ -396,6 +399,7 @@ def _as_records(stream: str, data: List[Dict[str, Any]]) -> List[AirbyteMessage]
     return [_as_record(stream, datum) for datum in data]
 
 
+# TODO: Replace call of this function to fixture in the tests
```

Comment:
nit: remove old todo`s

Reply:
This is a new TODO. I have added a fixture but haven't replaced this function yet. We need to replace it in the follow-up PR.